src/useClickOutside/__docs__/story.mdx

```diff
@@ -35,7 +37,7 @@ export function useClickOutside<T extends HTMLElement>(
 
 #### Arguments
 
-- **ref** _`RefObject<T> | MutableRefObject<T>`_ - React ref object with target HTML element.
-- **callback** _`EventListener`_ - Callback that will be triggered during the click.
-- **events**: _`string[]`_ _(default: `['mousedown', 'touchstart']`)_ - Events list that will be
-  used as triggers for outside click.
+- **ref** _`RefObject<T> | MutableRefObject<T>`_ - React ref object containing the target HTML element.
```

Comment:
Do ref objects "contain" or "reference" the target HTML element?

Reply:
I think "contain" is better. The React ref object is an object which contains something in its `current` property.